deepzoom/__init__.py

```diff
@@ -434,7 +434,7 @@ def create(self, source, destination):
                     jpeg_quality = int(self.image_quality * 100)
                     tile.save(tile_file, "JPEG", quality=jpeg_quality)
                 else:
-                    tile.save(tile_file)
+                    tile.save(tile_file, self.descriptor.tile_format)
```

Comment:
According to the docs, this should be inferred from the file path. I will change it to accept the output path instead of passing in the format:
![Uploading image.png…]()